src/main/resources/mentorshipFaqPage.json

```diff
@@ -0,0 +1,57 @@
+{
+  "id": "page:MENTORSHIP_FAQ",
+  "heroSection": {
+    "title": "Mentorship FAQ"
+  },
+  "commonFaqSection": {
+    "title": "Mentors and Mentees FAQ",
+    "items": [
+      {
+        "title": "How do I join the mentorship program?",
```

Comment:
Could you already include all the existent data from the current website? Because I don't think it will change much.